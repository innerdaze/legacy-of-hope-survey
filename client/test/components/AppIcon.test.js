import React from 'react'
import AppIcon from '../../components/AppIcon'

let wrapper

describe('Testing on AppIcon...', () => {
  test('Expect AppIcon to have a SVGIcon comp', () => {
    wrapper = shallow(<AppIcon />)

    expect(wrapper.find('SVGIcon')).toHaveLength(1)
    expect(wrapper).toMatchSnapshot()
  })
})
