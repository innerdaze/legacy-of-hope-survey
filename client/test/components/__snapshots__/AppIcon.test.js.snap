// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing on AppIcon... Expect AppIcon to have a SVGIcon comp 1`] = `
<SVGIcon
  a11yTitle="App Icon"
  type="logo"
  version="1.1"
  viewBox="0 0 491 49"
>
  <defs>
    <path
      d="M159.163464,0.858126387 L81.2757707,31.5580333 C79.5505133,32.3538914 79.1835645,32.7960089 78.9403021,33.2568424 C78.6936766,33.7235393 78.4732083,34.7022474 78.4732083,36.6544606 L78.4732083,123.17238 L78.4732083,125.59925 L76.3712865,126.691342 L2.21997582,163.337119 C0.0620027696,164.648018 0.00146742167,165.242115 0.00146742167,168.19086 L0.00146742167,259.683863 C0.00146742167,260.973016 0.953964963,262.621735 2.10338922,263.203019 L78.8233418,302.032941 C81.1296638,303.398201 82.3882012,303.521292 84.5431848,302.638687 L159.044635,264.900663 L160.679463,264.051453 L162.314291,264.900663 L237.049289,302.759837 C238.31941,303.521292 239.875019,303.039412 242.186573,302.273686 L319.257039,263.201077 C320.40011,262.6229 321.367929,261.002915 321.35896,259.681922 L321.35896,168.067769 C321.35896,166.252082 321.086178,165.538388 320.774906,165.033987 C320.468493,164.538517 319.858282,163.910637 318.439438,163.092491 L244.989135,126.810744 L242.887213,125.718652 L242.887213,123.291782 L242.887213,36.4098321 C242.924581,35.4898347 242.451134,34.176335 241.719479,33.1337516 C241.231708,32.438696 240.570927,31.872491 239.737133,31.4351366 L161.965653,0.85657319 C161.364036,0.56845517 161.164494,0.505162397 160.916373,0.492736822 C160.533356,0.474875058 160.078594,0.597189312 159.163464,0.858126387 Z M160.683574,8.62430491 L161.967895,9.10967893 L220.938665,32.407632 L229.346353,35.6839067 L221.172212,39.5668989 L162.201815,67.4759052 L160.683574,68.2039662 L159.165332,67.4759052 L100.078349,39.5668989 L91.9038347,35.6839067 L100.311522,32.407632 L159.399253,9.10967893 L160.683574,8.62430491 L160.683574,8.62430491 Z M85.9485764,41.265708 L91.2035678,43.8139216 L154.728316,73.907111 L156.946824,74.8778591 L156.946824,77.4260727 L156.946824,155.449986 L156.946824,161.63831 L151.575246,158.968753 L88.0504983,126.81288 L85.9485764,125.720788 L85.9485764,123.293918 L85.9485764,47.3328832 L85.9485764,41.265708 Z M235.418571,41.265708 L235.418571,47.3328832 L235.418571,123.293918 L235.418571,125.720788 L233.316649,126.81288 L169.791902,158.968753 L164.420324,161.63831 L164.420324,155.449986 L164.420324,77.5474162 L164.420324,74.9992026 L166.638832,74.0284545 L230.16358,43.8139216 L235.418571,41.265708 Z M82.0948663,132.273337 L83.7300681,133.122742 L143.751239,163.579806 L150.874605,167.2205 L143.634653,170.618118 L83.7300681,199.133648 L82.2118266,199.861709 L80.6935851,199.133648 L19.8548206,170.253893 L12.4979075,166.856275 L19.738234,163.21597 L80.4604119,133.122742 L82.0948663,132.273337 Z M239.155321,132.273337 L240.790149,133.122742 L301.512335,163.21597 L308.752287,166.856275 L301.395748,170.253893 L240.673563,199.133648 L239.155321,199.861709 L237.63708,199.133648 L177.732495,170.618118 L170.492542,167.2205 L177.615909,163.579806 L237.520493,133.122742 L239.155321,132.273337 Z M7.47683554,172.923451 L12.731828,175.471858 L76.2565683,205.565048 L78.4750767,206.535796 L78.4750767,209.083815 L78.4750767,287.107884 L78.4750767,293.296208 L73.1034988,290.626651 L9.57875734,258.349279 L7.47683554,257.257381 L7.47683554,254.830511 L7.47683554,178.990626 L7.47683554,172.923451 L7.47683554,172.923451 Z M156.946824,172.923451 L156.946824,178.990626 L156.946824,254.830511 L156.946824,257.257381 L154.844902,258.349279 L91.3201544,290.626651 L85.9485764,293.296208 L85.9485764,287.107884 L85.9485764,209.083815 L85.9485764,206.535796 L88.1670848,205.565048 L151.691833,175.471858 L156.946824,172.923451 L156.946824,172.923451 Z M164.420324,172.923451 L169.675315,175.471858 L233.200063,205.565048 L235.418571,206.535796 L235.418571,209.083815 L235.418571,287.107884 L235.418571,293.296208 L230.046993,290.626651 L166.522246,258.470816 L164.420324,257.378531 L164.420324,254.95166 L164.420324,178.990626 L164.420324,172.923451 Z M313.890319,172.923451 L313.890319,178.990626 L313.890319,254.830511 L313.890319,257.257381 L311.788397,258.349279 L248.263649,290.626651 L242.892071,293.296208 L242.892071,287.107884 L242.892071,209.083815 L242.892071,206.535796 L245.110579,205.565048 L308.635327,175.471858 L313.890319,172.923451 L313.890319,172.923451 Z"
      id="cubes-white-a"
    />
  </defs>
  <g
    fill="none"
    fillRule="evenodd"
    transform="translate(2 2)"
  >
    <g
      transform="translate(82.324 91.746)"
    >
      <use
        fillRule="nonzero"
        stroke="#FFF"
        strokeWidth="1.5"
        xlinkHref="#cubes-white-a"
      />
    </g>
    <ellipse
      cx="243.5"
      cy="244.5"
      rx="243.5"
      ry="244.5"
      stroke="#FFF"
      strokeWidth="3"
    />
  </g>
</SVGIcon>
`;
